pages/common/kak.md

```diff
@@ -0,0 +1,36 @@
+# kak
+
+> Kakoune is a mode-based code editor implementing the "multiple selections" paradigm.
+> Data can be selected and simultaneously edited in different locations, using multiple selections; users can also connect to the same session for collaborative editing.
+
+- Open a file and enter normal mode, to execute commands:
+
+`kak {{file}}`
+
+- Enter insert mode from normal mode, to write text into the file:
+
+`i`
+
+- Escape insert mode, to go back to normal mode:
+
+`<Escape>`
+
+- Replace all instances of "foo" in the current file with "bar":
+
+`%s{{foo}}<Enter>c{{bar}}<Escape>`
+
+- Un-select all secondary selections, and keep only the main one:
+
+`<Space>`
+
+- Search for numbers and select the first two:
+
+`/\d+<Enter>N`
+
+- Insert the contents of a file:
+
+`!cat {{/etc/issue}}<Enter>`
```

Comment:
I assume this is an example path? Might be best to change it to `{{path/to/file}} as per the style guide`. 👍

Reply:
If you use `path/to/file`, please use it in the first example too, which just has `file`. Otherwise, use `file` in both cases.

Reply:
I followed the convention from the style-guide, both examples now use `{{path/to/file}}`.